UITests/AuthenticationManagerTests.swift

```diff
@@ -148,14 +122,11 @@ class AuthenticationManagerTests: KIFTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
-        setPasscode("1337", interval: .Immediately)
+        PasscodeUtils.setPasscode("1337", interval: .Immediately)
 
         openAuthenticationManager()
         tester().tapViewWithAccessibilityLabel("Require Passcode, Immediately")
 
-        tester().waitForViewWithAccessibilityLabel("Enter Passcode")
-        enterPasscodeWithDigits("1337")
-
```

Comment:
So interestingly enough, this happens when the KIFTestActor taps on the Menu button. Looks like we're dividing by zero when setting up the height constraint for the menu here [1]. Not sure how this is happening though. Might be early enough in the lifecycle that nothing is setup yet?

@fluffyemily Have you seen this before?

[1] https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Widgets/Menu/MenuView.swift#L330